### DISCUSSION TITLE
Victron IP22 VE.Direct Anschluss

### DISCUSSION THREAD

**Discussion by iroc-fabi** [hoylabs/OpenDTU-OnBattery#1061] [Q&A]
Ich nutze das Ladegerät Victron IP22 24V/16A.
Im Gehäuse befinden sich fünf Kontakte, die man für einen VE.Direct Anschluss nutzen kann.
Siehe die Projekte dazu:
[wasn-eu](https://github.com/wasn-eu/Victron_BlueSmart_IP22)
[pvtex](https://github.com/pvtex/Victron_BlueSmart_IP22)

Ich nutze für "openDTU on Battery" die OpenDTU FusionOpenDTU Fusion mit dem [CAN/ISO Shield](https://shop.allianceapps.io/products/opendtu-fusion-can-iso-shield)
Kann ich da direkt das Ladegerät anschließen?
Wird das von DTU on Battery erkannt?

**Comment by SW-Niko** [comment]
Hallo @iroc-fabi,
die Victron IP-Ladegeräte werden meines Wissens nicht direkt von OpenDTU-OnBattery unterstützt.
Du müsstest die IP-Ladegeräte der OpenDTU-OnBattery als MPPT oder als SmartShunt unterjubeln und hoffen das einige Daten die über VE.Direct kommen das gleiche Format haben. 

Ich verwende keine IP-Ladegeräte und kann dir hier nicht wirklich helfen. Vielleicht meldet sich noch jemand der damit Erfahrung hat.
